src/aave-v2/MorphoStorage.sol

```diff
@@ -69,7 +69,7 @@ abstract contract MorphoStorage is OwnableUpgradeable, ReentrancyGuardUpgradeabl
     IEntryPositionsManager public entryPositionsManager;
     IExitPositionsManager public exitPositionsManager;
     IInterestRatesManager public interestRatesManager;
-    address public incentivesVault; // Deprecated.
+    address internal incentivesVault; // Deprecated.
```

Comment:
I don't think we should do this because it's changing the ABI

Reply:
Yes I agree, I will remove the commit